vividus-plugin-mobile-app/src/main/java/org/vividus/mobileapp/steps/TouchSteps.java

```diff
@@ -123,7 +123,8 @@ public void swipeToElement(SwipeDirection direction, Locator locator, Duration s
         locator.getSearchParameters().setWaitForElement(false);
         Supplier<Rectangle> swipeArea = createSwipeArea();
         List<WebElement> elements = new ArrayList<>(searchActions.findElements(locator));
-        if (elements.isEmpty())
+        if (elements.isEmpty() || !isElementInteractable(elements.get(0).getRect(), swipeArea.get(),
```

Comment:
there is no sense anymore to keep `swipeArea` as Supplier

Reply:
done